### ISSUE TITLE
Any way to disable it on startup?

### ISSUE THREAD

**Issue by inside83** [joaopaulovieira/clappr-context-menu-plugin#10]
I've implemented th plugin but the context menu is opened by default when a video starts.
How to make it closed by default and opened on right click only?

Also, how to style it to look like on the demo image, with semi-transparent background?

Thanks

**Comment by joaopaulovieira** [comment]
Hi @inside83! I can't reproduce this behavior, this still occurs? I created one [demo page](https://joaopaulovieira.github.io/clappr-context-menu-plugin/) using the latest version of the plugin, can you reproduce the reported behavior on this page?

About the custom style, you can use the `rgba` function to add one opacity into the `background-color` property. Just configure the option `contextMenu.customStyle.list`with your `background-color`.

One example:

```javascript
var playerElement = document.getElementById("player-wrapper");

var player = new Clappr.Player({
  source: 'http://clappr.io/highline.mp4',
  poster: 'http://clappr.io/poster.png',
  height: 360,
  width: 640,
  // playback: { controls: true }, // use this config when test with the @clappr/core
  plugins: [ContextMenuPlugin],
  contextMenu: {
    menuItems: ['copyURL', 'copyURLCurrentTime', 'loop', 'playerVersion'],
    extraOptions: [
      {
        name: 'test',
        label: 'Test Label',
        callback: function() {
          console.log('A Absolutely awesome extra context menu item action');
        }
      }
    ],
    customStyle: {
    list: {
        'background-color': "rgba(" + 0 + "," + 0 + "," + 0 + "," + 0.3 + ")",
      },
      items: {
        'color': 'yellow'
      }
    }
  }
});

player.attachTo(playerElement);
```

**Comment by inside83** [comment]
Hey @joaopaulovieira 
Thank you for the swift reply.
That neat trick with rgb works 👍😀
I'm using 
`https://cdn.jsdelivr.net/npm/clappr@latest/dist/clappr.min.js`
not
`https://cdn.jsdelivr.net/npm/@clappr/player@latest/dist/clappr.min.js`
like you in the demo because, unfortunately, chromecast plugin doesn't work with the latest Clappr version

But even If I switch to `clappr/player` and disable chromecast plugin, the context menu is still, by default, open.

Your demo seams to be ok though 🤷‍♂️

**Comment by inside83** [comment]
PLugin reference should go under 'core', right?

**Comment by joaopaulovieira** [comment]
Yes. I'll try to test on the older versions of Clappr and let you know if I discover anything

**Comment by inside83** [comment]
Please do, it's super annoying.
Thank you 🙏

**Comment by joaopaulovieira** [comment]
Can you tell in which environment (Clappr version/O.S/Browser and version/Running with any specific framework) you reproduce this behavior? Because I tested it on old versions of Clappr and I still couldn't reproduce the problem x.x

**Comment by inside83** [comment]
You can't? 😳
I'm testing on Chrome 88 / Windows 10
Could I send you my setup and link in private please?
